File: swarm-keydb-js/examples/user-profile.mjs
import { SwarmKeyDb } from '../index.js';

const db = new SwarmKeyDb({ host: '127.0.0.1', port: 6379 });
await db.connect();
await db.put('user:42', JSON.stringify({ name: 'Ada', role: 'admin' }));
console.log(await db.get('user:42'));
await db.disconnect();

File: swarm-keydb-py/examples/user_profile.py
from swarm_keydb import SwarmKeyDb

client = SwarmKeyDb(host="127.0.0.1", port=6379)
client.put("user:42", '{"name":"Ada","role":"admin"}')
try:
    print(client.get("user:42"))
except Exception as exc:
    print(f"Failed to read user:42: {exc}")
